tasmota/xdrv_05_irremote_full.ino

```diff
@@ -412,7 +417,17 @@ uint32_t IrRemoteCmndIrHvacJson(void)
     }
   }
 
-  if (val = root[PSTR(D_JSON_IRHVAC_MODEL)]) { state.model = IRac::strToModel(val.getStr()); }
+  bool stateOnly = false; // flag that means to update the state of IRremote without sending actual IR command
+  // parse model of the ac
+  if (val = root[PSTR(D_JSON_IRHVAC_MODEL)]) {
```

Comment:
I'm not a big fan of changing the semantic of an existing field. I would rather have a "StateOnly":true field instead.

Reply:
I totally agree. Just not sure about the right processes here to add a new field. Any guidelines? Is this trivial? Should I just try to do it or do you prefer to write this part yourself? Will you accept PRs that add fields, options, etc?
Overall I understand the IR/AC part well but more cautious about the Tasmota part.

Reply:
Just add a new field. As it's JSON it's position isn't a problem and ignoring the field shouldn't be a problem either.